geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/ErrorMode.scala

```diff
@@ -8,9 +8,41 @@
 
 package org.locationtech.geomesa.convert
 
-object ErrorMode extends Enumeration {
+import org.locationtech.geomesa.utils.conf.GeoMesaSystemProperties.SystemProperty
+
+object ErrorMode extends Enumeration with Modes {
```

Comment:
rename file Modes.scala and move that trait up top?

Reply:
Done